components/bsx/Create/CreateCollection.vue

```diff
@@ -101,6 +104,38 @@ export default class CreateCollection extends mixins(
     }
     return ''
   }
+  async getFeesToken() {
+    try {
+      const api = await this.useApi()
+      const tokenId = await getAssetIdByAccount(api, this.accountId)
+      if (tokenId === '0') {
+        // use token different from BSX
+        this.useKSM = false
+      } else {
+        this.useKSM = true
+      }
+    } catch (e) {
+      this.$consola.log(e)
+    }
+  }
+
+  @Watch('accountId', { immediate: true })
+  onAccountIdChange(val: string, oldVal: string) {
+    if (shouldUpdate(val, oldVal)) {
+      this.getFeesToken()
+    }
+  }
+
+  KsmToBsx(KsmValue: number) {
```

Comment:
please follow eslint rules: function and parameter names starts with small letter

Reply:
sure thing. my bad